CHANGES.md

```diff
@@ -10,6 +10,8 @@ To be released.
 
  -  The `uri` type has completly gone; use `url` instead.
     [[#126], [#281] by Jonghun Park]
+ -  Fixed locale to open package.toml.
```

Comment:
It would be if the changelog specifies the actual behavior it fixed, e.g.:

> Fixed a bug that *package.toml* file had been failed to load when the system locale does not use UTF-8.  It's now independent from the system locale settings.

Reply:
Sure, I'll fix the changelog and rebase it!